testcases-sparql-1.2/aggregates/agg-empty-group-1.srx

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<sparql xmlns="http://www.w3.org/2005/sparql-results#">
+	<head>
+		<variable name="max"/>
+	</head>
```

Comment:
Needs `<variable name="x">` because there are two variables in `SELECT ?x (... AS ?max)`

The origin SPARQL test suite is wong in the HTML page and right in the actual files : `agg-empty-group.srx` vs `agg-empty-group2.srx` [[rdf-tests:data-sparql11/aggregates/agg-empty-group2.srx](https://github.com/w3c/rdf-tests/blob/gh-pages/sparql11/data-sparql11/aggregates/agg-empty-group2.srx)], same as [w3c:data-sparql11/aggregates/agg-empty-group2.srx](https://www.w3.org/2009/sparql/docs/tests/data-sparql11/aggregates/agg-empty-group2.srx)).

Reply:
> Needs <variable name="x"> because there are two variables in SELECT ?x (... AS ?max)

Good point, will fix.

> The origin SPARQL test suite is wong in the HTML page and right in the actual files : agg-empty-group.srx vs agg-empty-group2.srx [rdf-tests:data-sparql11/aggregates/agg-empty-group2.srx], same as w3c:data-sparql11/aggregates/agg-empty-group2.srx).

Ah, interesting. I'm working off a local copy in the RDF4J testsuite, which has this mistake in the actual files. So I guess this was redacted at some point after I created the copy.